GOVERNANCE.md

```diff
@@ -150,10 +146,19 @@ Once a PR is ready to be landed, the CPC member who lands the pull request shoul
 * Send a notification to the project contacts for the project identified in the PR
   indicating that a new Regular CPC member has joined the CPC on behalf of the project.
 * Add the member to the github `cpc-regular-members` [team][cpc regular members team]
-* Add the member to the `cpc-private` email list and directory by opening a PR against the [OpenJS Foundation directory](https://github.com/openjs-foundation/directory-private/blob/master/groups/cross-project-council.yml)
+* Add the member to the `cpc-private` email list and private directory by opening a PR against the [OpenJS Foundation CPC directory][]
 * Introduce the new member at the next CPC meeting.
 
-Former Voting members whose terms have just ended will automatically become Regular members, unless they indicate otherwise.
+_Note: Former Voting members whose terms have just ended will automatically become Regular members, unless they indicate otherwise._
+
+## Definition of an Active OpenJS Collaborator
+
+### To be considered An active OpenJS Collaborator, one of the following is required:
+
+- An active member of a project community or collaboration space which is defined as having recent, sustained contributions to the project during the past 90 days.
+- A demonstrated level of contribution to the CPC's work (meetings, issues, pull-requests) as an observer during the past 30 days.
+
+_Note: Contributions can include, but are not limited to, participation in: meetings, issues or pull-requests._
```

Comment:
```suggestion
_Note: Contributions can include, but are not limited to, participation in meetings, issues or pull-requests, editing documentation, community management, marketing, organizing events, etc._
```

Reply:
Do you think that gets too fuzzy? Do we need to call out "OpenJS Events" instead of just "events" for example? Or do you think that is implied enough within the context?

Reply:
Perhaps something along the lines of:
>_Note: Contributions can include, but are not limited to, participation in meetings, issues or pull-requests, editing documentation, community management, marketing, organizing events, as well as similar activities as they relate to the OpenJS Foundation._